src/courses/beginner/08.md

```diff
@@ -139,7 +129,7 @@ When this flag is passed, the control level status outcomes of the profile execu
 
```

Comment:
I think that the enhanced outcomes info should be in its own separate subsection.  not a 'details' tab underneath the reporter flags.  it should also be a little bit longer to elaborate what the difference is before/after (specifically before = pass/fail/err only whereas we likely also want to see the NA and NR separated out too).  we should also say that most of our example inspec exec invocations don't use it due to the simplicity of the controls and in order to manage the length of the lines that need to be copy pasted, but in real world circumstances we'd expect folks to use this flag.